doc/en/developer/source/qa-guide/index.rst

```diff
@@ -1,13 +1,15 @@
 Automatic Quality Assurance checks
 ==================================
 
-The GeoServer builds on Github Actions and `https://build.geoserver.org/ <https://build.geoserver.org/>`_ apply
+The GeoServer builds on Github Actions and `https://build.geoserver.org/ <https://build.geoserver.org/>`__ apply
 `PMD <https://pmd.github.io/>`_ and `Error Prone <https://errorprone.info/>`_ checks on the code base
 and will fail the build in case of rule violation.
 
-In case you want to just run the build with the full checks locally, use the following command::
+In case you want to just run the build with the full checks locally, use the following command:
 
-    mvn clean install -Dqa -Dall
+.. code-block:: bash
+
+   mvn clean install -Dqa -Dall
```

Comment:
I realise it was existing, but does `-Dall` make sense here? Isn't that a geotools thing?

Reply:
Whoops yeah, copy/paste error, should be ``-Prelease``